### DISCUSSION TITLE
Linux client

### DISCUSSION THREAD

**Discussion by ghost** [navidrome/navidrome#2440] [Ideas]
Hello, is there any wip for a linux client, it would be great to stream music from my navidrome server to my linux phone.

**Comment by metalheim** [comment]
You can try feishin https://github.com/jeffvli/feishin

**Comment by dweymouth** [comment]
I have no idea if there are any Subsonic clients that support Linux Mobile - but you could use the Navidrome web UI as a PWA maybe (does Linux mobile support Progressive Web Apps?)

**Reply by certuna** [reply]
Chrome supports PWAs

**Comment by yllekz** [comment]
You can install Firefox with the "[Progressive Web Apps for Firefox](https://addons.mozilla.org/en-US/firefox/addon/pwas-for-firefox/)" extension to make a PWA of Navidrome. I do this for Linux and macOS and it works perfectly.
